src/components/RoomView.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { setRoomId } from "../store/playerSlice";
+import { useParams } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+import { onValue, ref, set } from "firebase/database";
+import { database } from "../utils/firebase";
+import { setAllPlayers } from "../store/allPlayersSlice";
+import RedTeamBox from "./teamBoxes/TeamOneBox";
+import BlueTeamBox from "./teamBoxes/TeamTwoBox";
+const RoomView = () => {
```

Comment:
i think this file might need to be deleted. this is the roomview before the refactor and might cause some issues in main